### DISCUSSION TITLE
modal with checkbox in table

### DISCUSSION THREAD

**Discussion by devlzcode** [adobe/react-spectrum#7062] [Q&A] [answered]
I have a table and on each row are a set of actions which open a modal. One of these modals are a form and contain a checkbox which is causing this error: `Error: A slot prop is required. Valid slot names are "selection".`
How do I fix this other than moving the modal out of the table and handling its open sate manually?

Example:
```tsx
  <Cell>
    <DialogTrigger>
    <Button/>
    <Modal>
      <Dialog>
        <Checkbox/>
      <Dialog/>
    </Modal>
  </DialogTrigger>
<Cell>
```

**Comment by snowystinger** [comment]
We should be able to fix this by clearing the context in the Cells. In the meantime, you can clear it yourself. Just make a checkbox context that's empty.
```
<Provider values={[
  [CheckboxContext, {}]
]}
```

**Reply by devlzcode** [reply, marked as answer]
> We should be able to fix this by clearing the context in the Cells. In the meantime, you can clear it yourself. Just make a checkbox context that's empty.
> 
> ```
> <Provider values={[
>   [CheckboxContext, {}]
> ]}
> ```

Setting slot to `null` specifically works as well.
